src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -72,6 +78,12 @@
      */
     private boolean ignoreOverridden;
 
+    /** The visibility scope where methods are checked. */
+    private Scope scope = Scope.ANONINNER;
```

Comment:
@rnveach Let me know if I get it right: is your suggestion to use `null` as a default both for `scope` and `excludeScope`? And the meaning would be:
- `scope=null`: check all scopes
- `excludeScope=null`: exclude no scopes
Is it correct?